api/storage_service.cc

```diff
@@ -644,6 +644,23 @@ void set_storage_service(http_context& ctx, routes& r, sharded<service::storage_
         });
     });
 
+    ss::perform_keyspace_offstrategy_compaction.set(r, wrap_ks_cf(ctx, [] (http_context& ctx, std::unique_ptr<request> req, sstring keyspace, std::vector<sstring> tables) -> future<json::json_return_type> {
+        std::unordered_map<sstring, bool> needed;
+        for (const auto& t : tables) {
+            needed[t] = false;
+        }
```

Comment:
Can write `bool needed = false;`

Reply:
We can return a bool rather than a count

Reply:
fixed in v3